pkg/migration/controllers/migration.go

```diff
@@ -211,8 +211,50 @@ func (m *MigrationController) handle(ctx context.Context, migration *stork_api.M
 		return nil
 	}
 
+	// TODO: verify if appActivated flags need to be set as default
 	migration.Spec = setDefaults(migration.Spec)
 
+	if migration.GetAnnotations() != nil {
+		remoteConfig, err := getClusterPairSchedulerConfig(migration.Spec.ClusterPair, migration.Namespace)
+		if err != nil {
+			return err
+		}
+		remoteOps, err := storkops.NewForConfig(remoteConfig)
+		if err != nil {
+			m.recorder.Event(migration,
+				v1.EventTypeWarning,
+				string(stork_api.MigrationStatusFailed),
+				err.Error())
+			return nil
+		}
+		// get remote cluster migration schedule object
+		migrSched, err := remoteOps.GetMigrationSchedule(migration.GetAnnotations()[StorkMigrationName], migration.Namespace)
```

Comment:
Why not just use GetMigrationSchedule(migration.Name, migration.Namespace) since the names are the same on both source and DR side?

Reply:
We just don't need the StorkMigrationName annotation?

Reply:
controller is trying to get migration-schedule object and not `migration`, I added separate annotation to avoid confusion